tests/unit_node/worker_threads_test.ts

```diff
@@ -16,7 +16,7 @@ Deno.test("[node/worker_threads] BroadcastChannel is exported", () => {
 });
 
 Deno.test("[node/worker_threads] MessageChannel are MessagePort are exported", () => {
-  assertEquals<unknown>(workerThreads.MessageChannel, MessageChannel);
```

Comment:
Why this change?

Reply:
Because the MessageChannel exported from worker_threads is now NodeMessageChannel